lib/ddtrace/configuration/settings.rb

```diff
@@ -4,7 +4,7 @@
 require 'ddtrace/ext/analytics'
 require 'ddtrace/ext/distributed'
 require 'ddtrace/ext/profiling'
-require 'ddtrace/ext/runtime'
+# require 'datadog/core/ext/environment'
```

Comment:
`FALLBACK_SERVICE_NAME` is in `ddtrace/ext/environment` now, this `require` needs to be updated. I'm guessing the tests still pass because that file ends up being required elsewhere (?).

Reply:
Fixed it.